modules/core/04-channel/keeper/packet.go

```diff
@@ -33,6 +33,13 @@ func (k Keeper) SendPacket(
 		return 0, errorsmod.Wrap(types.ErrChannelNotFound, sourceChannel)
 	}
 
+	if channel.FlushStatus != types.NOTINFLUSH {
+		return 0, errorsmod.Wrapf(
+			types.ErrInvalidChannelState,
+			"channel should not be in %s state", types.NOTINFLUSH.String(),
```

Comment:
do we prefer being more explicit here, i.e:

```suggestion
return 0, errorsmod.Wrapf(types.ErrInvalidFlushStatus, "expected flush status to be %s during packet send, got %s", types.NOTINFLUSH, channel.FlushStatus)
```

Reply:
ah I just remembered I'm adding that error in the ChanUpgradeOpen pr

Reply:
Agreed